scripts/mirror.js

```diff
@@ -428,15 +499,11 @@ const bumpVersion = (sourceData, originalData, destination, targetVersion) => {
   }
 
   if (Array.isArray(sourceData)) {
-    newData = [];
-    for (let i = 0; i < sourceData.length; i++) {
-      newData[i] = bumpVersion(
-        sourceData[i],
-        originalData,
-        destination,
-        targetVersion,
-      );
-    }
+    newData = combineStatements(
+      ...sourceData.map((d) =>
+        bumpVersion(d, originalData, destination, targetVersion),
```

Comment:
I'd avoid single-char variables:

```suggestion
      ...sourceData.map((data) =>
        bumpVersion(data, originalData, destination, targetVersion),
```